packages/vega-scenegraph/src/util/text.js

```diff
@@ -129,10 +129,14 @@ export function offset(item) {
   // perform our own font baseline calculation
   // why? not all browsers support SVG 1.1 'alignment-baseline' :(
   var baseline = item.baseline,
-      h = fontSize(item);
+      h = fontSize(item),
+      lh = lineHeight(item);
```

Comment:
This `lineHeight` call could be inlined to avoid invoking it even when it is not needed.